src/main/java/edu/illinois/cs/dt/tools/minimizer/TestMinimizer.java

```diff
@@ -97,7 +102,16 @@ public MinimizeTestsResult run() throws Exception {
             List<PolluterData> polluters = new ArrayList<>();
             int index = 0;
 
-            if (ONE_BY_ONE_POLLUTERS) {
+            if (CUSTOM_POLLUTERS != "") {
+                final List<String> customOrder = new ArrayList<String>(Arrays.asList(CUSTOM_POLLUTERS.split(";")));
+                if (ONE_BY_ONE_POLLUTERS) {
```

Comment:
Since a lot of the logic for dealing with ONE_BY_ONE_POLLUTERS is the same, just that starting order is customOrder, can it be somehow merged with the remaining logic, like setting some initial order to be either customOrder or fullTestOrder/testOrder depending on the CUSTOM_POLLUTERS option?

Reply:
Yeah, that is a good point. I've merged the logic as you suggested. Please check if it looks good to you now.